pages/admin_pages.py

```diff
@@ -25,11 +25,11 @@ def choose_study():
         return redirect('/view_study/{:d}'.format(allowed_studies.values_list('pk', flat=True).get()))
 
     # Otherwise, show the "Choose Study" page
-    allowed_studies_json = Study.query_set_as_unpacked_native_json(allowed_studies)
+    allowed_studies_list = Study.query_set_as_unpacked_native_python(allowed_studies)
     return render_template(
         'choose_study.html',
-        studies=allowed_studies_json,
-        allowed_studies=allowed_studies_json,
+        studies=allowed_studies_list,
+        allowed_studies=get_researcher_allowed_studies(),
```

Comment:
Can we use `allowed_studies=allowed_studies_list` instead of `allowed_studies=get_researcher_allowed_studies()`? Just asking because that's what the original code says it's doing.

Reply:
Those methods expose slightly different amounts of information about studies but otherwise function the same. The allowed_studies_list contains all of the study fields except timestamps ('created_on' and 'last_updated') and is used only in this context. The get_researcher_allowed_studies method gives only the "name", "object_id", "id" and "is_test" fields. Another 5 or so pages in Beiwe also use the 'allowed_studies=get_researcher_allowed_studies()' line because its used by the navbar. 

My inclination would be to leave this as it is currently written in the interest of consistency and ease of future maintenance, though it is slightly less efficient. If we make future changes to either the navbar or the 'get_researcher_allowed_studies()' function I would expect them to appear on all pages that use the navbar, but changes specific to the choose_study page can make use of another function that is already differentiated.

Reply:
I agree with the consistency and ease of future maintenance.